src/test/unit/symbol-store.test.js

```diff
@@ -119,7 +119,6 @@ describe('SymbolStore', function() {
     );
     expect(symbolProvider.requestSymbolsFromServer).toHaveBeenCalledTimes(2);
     expect(symbolProvider.requestSymbolTableFromAddon).toHaveBeenCalledTimes(2);
-    expect(console.log).toHaveBeenCalledTimes(2); // Once for each uncached call
```

Comment:
I'd like to get test coverage over the new methods if we can.

Reply:
Not sure I understand - which new methods?

Reply:
Sorry, I used the words "methods" and meant "new lines of code". Since this bug is about ensuring that the error messages are properly surfaced to the user, I'd like to ensure we exercise the error paths in tests, so that we ensure that they get surfaced to the user.

The current implementation removes the coverage over the console log messages, but doesn't add coverage for the new errors as far as I can see.